src/blas/backends/cublas/CMakeLists.txt

```diff
@@ -42,7 +42,10 @@ set_target_properties(${LIB_OBJ} PROPERTIES
   POSITION_INDEPENDENT_CODE ON)
 
 target_link_libraries(${LIB_NAME} PUBLIC ${LIB_OBJ})
-
+if (USE_ADD_SYCL_TO_TARGET_INTEGRATION)
+  add_sycl_to_target(TARGET ${LIB_NAME})
+  add_sycl_to_target(TARGET ${LIB_OBJ})
```

Comment:
This will work in hipSYCL, but I think it won't work for ComputeCpp which additionally needs the source files in  `SOURCES`. Can the sources be easily added? If not, I would suggest to just special-case this for hipSYCL instead of something generic like `USE_ADD_SYCL_TO_TARGET_INTEGRATION`.

Reply:
Well, this seems to be easy at some places I have to have a closer look.

Reply:
Okay, this turned out to be rather straightforward :)